packages/tpetra/core/test/CrsMatrix/CMakeLists.txt

```diff
@@ -88,17 +88,15 @@ TRIBITS_ADD_EXECUTABLE_AND_TEST(
   STANDARD_PASS_OUTPUT
   )
 
-IF(NOT Tpetra_INST_HIP)
-  # This test causes AMD MI25 to permanently crash (until reboot)
-  TRIBITS_ADD_EXECUTABLE_AND_TEST(
-    CrsMatrix_2DRandomDist
-    SOURCES
-      CrsMatrix_2DRandomDist.cpp
-    COMM serial mpi
-    PASS_REGULAR_EXPRESSION "PASS"
-    FAIL_REGULAR_EXPRESSION "FAIL"
-    )
-ENDIF()
+# This test causes AMD MI25 to permanently crash (until reboot)
```

Comment:
Do we still need this comment if we are enabling the test?  
Do we know why this test crashed the AMD?

Reply:
No, good catch, that comment doesn't apply to rocm 4.2 and later.

Reply:
Yes, this is a bug in the AMD software stack prior to ROCM 4.2, ROCM 4.2 is available at ORNL, and will be publicly released in the next couple weeks. We will shift minimum version of ROCM to 4.2 immediately with Kokkos since it fixes that really bad bug which has bothering us more than a year.